setup.py

```diff
@@ -39,8 +39,13 @@ def get_requirements():
         "Programming Language :: Python :: 3",
     ],
     install_requires=get_requirements(),
-    python_requires=">=3.8",
+    python_requires=">=3.9",
     project_urls={
         "Documentation": "https://release-engineering.github.io/exodus-gw",
     },
+    entry_points={
+        "console_scripts": [
+            "exodus_gw_alembic_upgrade = exodus_gw.alembic_upgrade:entry_point"
```

Comment:
Generally command-line tools prefer '-' over '_' to separate words (examples: ansible-playbook, ssh-keygen, gpg-agent).    This should be the same, so:

```suggestion
            "exodus-gw-alembic-upgrade = exodus_gw.alembic_upgrade:entry_point"
```

Reply:
console script updated to `exodus-gw-alembic-upgrade`